### ISSUE TITLE
Problem with Asset Definition in Summary (Documentation Issue)

### ISSUE THREAD

**Issue by nimit199** [hyperledger/composer#4680]
Link - 
https://hyperledger.github.io/composer/latest/systemns/01_summary.html

The entire page does not make sense.

I am trying to write very detailed permissions.acl file for an application. But I am unable to provide proper "org.hyperledger.composer.system.****" Resource access as I am unable to find their documentation. I found this link but it is not helpful cause the description part of the page is very wierd.

For Example,

```
        Namespace                              Name              Description
org.hyperledger.composer.system.Transaction | Transaction | A transaction named Transaction
```

This doesn't make sense and I know what its called I am wondering what does it do. The entire page is confusing like such.

Let me know if there is a more detailed place which can explain all  "org.hyperledger.composer.system.****" Resources.

**Comment by nimit199** [comment]
I have also Raised the Issue in Stackoverflow as the documentation recommends to.
[Stackoverflow Link](https://stackoverflow.com/questions/57709612/more-information-on-org-hyperledger-composer-system-as-a-resource-in-permisi)

**Comment by ryjones** [comment]
⚠️ ⚠️ ⚠️

As of the 29th August 2019, the Hyperledger Composer project is in deprecated status. None of the maintainers are actively developing new features. None of the maintainers are actively providing support via GitHub issues. However, if you wish to submit code changes via pull requests, these will be merged.

It is highly recommended that you use Hyperledger Fabric v1.4+ instead, which features significant improvements to the developer experience, including a new programming model.

More information available here: What's new in Hyperledger Fabric v1.4

⚠️ ⚠️ ⚠️
